### PR TITLE
profile page enhanced

### PR DESCRIPTION
**Pull request by Ankita297**
## What is the change?
profile page updated
## Related issue?
close: #861 

## How was it tested?
I have tested it in my local system
## Checklist:
Before you create this PR, confirm all the requirements listed below by checking the checkboxes `[x]`:

- [x] Have you followed the [Contribution Guidelines](https://github.com/ALPHAVIO/BlogSite/blob/master/CONTRIBUTING.md) while contributing.
- [x] Have you checked there aren't other open [Pull Requests](https://github.com/ALPHAVIO/BlogSite/pulls) for the same update/change?
- [x] Did you lint your code before making the Pull Request?
- [x] Have you made corresponding changes to the documentation?
- [x] Your submission doesn't break any existing feature.
- [x] Have you tested the code before submission?

## Screenshots
WordNook - Google Chrome 15-05-2021 10_37_09
WordNook - Google Chrome 15-05-2021 10_37_13

 or Video:
Add a screenshot or demo video if appropriate.
WordNook - Google Chrome 15-05-2021 10_36_09
WordNook - Google Chrome 15-05-2021 10_36_20

### CODE REVIEW

**Comment by Pranav016** [comment]
image
 these buttons would look good center aligned

File: public/css/styles.css [hunk 1]
```diff
@@ -1274,3 +1274,55 @@ svg {
...
 	box-shadow: 0px 7px 5px rgba(0, 0, 0, 0.11);
 }
+
+/* read-profile form form styling  */
+
+.edit-form{
+	padding-top: 5%;
+	padding-bottom: 5%;
+	padding-right: 0%;
+	padding-left: 0%;
```
Comment (Pranav016) [inline public/css/styles.css:1284]: write clean css, these 4 lines can be replaced with a single line, `padding: 5% 0;` 

Example- 
```
padding: 25px 50px;
top and bottom paddings are 25px
right and left paddings are 50px
```

File: views/useritems/read-profile.ejs [hunk 2]
```diff
@@ -1,64 +1,62 @@
 <%- include("../partials/header", {isAuthenticated, user}) -%>
-    <div class="container">
-        <h1>Edit Profile</h1>
+<div class="container-centre edit-profile">
+    <h1 class="heading-centre heading-bold normal-heading-font">Edit Profile</h1>
```
Comment (Pranav016) [inline views/useritems/read-profile.ejs:3]: Is `container-centre` or `heading-centre` or `heading-bold` even a valid bootstrap class ?

File: views/useritems/read-profile.ejs [hunk 3]
```diff
@@ -1,64 +1,62 @@
...
+                        <div class="col-md-12 col-sm-12 col-xs-12">
                             <input class="form-control" type="password" name="password" id="password">
                             <span class="password-icon pb-3 pr-3"><i class="fa fa-eye" id="eye" onclick="togglePassword()"></i></span>
                         </div>
                     </div>
                     <div class="form-group">
                         <label class="col-md-3 control-label"></label>
-                        <div class="col-md-8">
-                            <input type="submit" class="btnFrame button px-4" value="Save Changes">
+                        <div class="col-md-12 col-sm-12 col-xs-12">
```
Comment (Pranav016) [inline views/useritems/read-profile.ejs:56]: very bad bootstrap and css, instead of these 3 classes you could just use one class `col-12`

**Comment by Ankita297** [comment]
WordNook - Google Chrome 15-05-2021 14_47_18
WordNook - Google Chrome 15-05-2021 14_47_28
is this okay ?

File: views/useritems/read-profile.ejs [hunk 4]
```diff
@@ -1,64 +1,62 @@
...
                             <input class="form-control" type="text" for="email" name="email" value="<%= user.email %>">
                         </div>
                     </div>
                     <div class="form-group">
-                        <label class="col-lg-8 control-label" for="photo">Profile Picture(Leave empty if you don't want to update profile picture)</label>
-                        <div class="col-lg-8">
-                            <input type="file" name="photo">
+                        <label class="col-md-12 col-sm-12 col-xs-12 control-label profile-file" for="photo" >Profile Picture(Leave empty if you don't want to update profile picture)</label>
+                        <div class="col-md-12 col-sm-12 col-xs-12">
```
Comment (Pranav016) [inline views/useritems/read-profile.ejs:40]: `col-12`

**Review by Pranav016 — CHANGES_REQUESTED** [review]
I didn't review your previous PR this carefully, don't know how many mistakes were there in those

File: public/css/styles.css (hunk 1 above)
Comment (Ankita297) [reply public/css/styles.css:1284]: I will correct it

File: views/useritems/read-profile.ejs (hunk 2 above)
Comment (Ankita297) [reply views/useritems/read-profile.ejs:3]: these classes are already define in style.css . These are already used in compose page so i have copy it from there

File: views/useritems/read-profile.ejs (hunk 3 above)
Comment (Ankita297) [reply views/useritems/read-profile.ejs:56]: okay Sir

**Comment by Ankita297** [comment]
done the changes

**Review by Pranav016 — APPROVED** [review]

**Pull request merged**